src/docs/upgrading.dox

```diff
@@ -1,5 +1,34 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+@section version_1120 Upgrading to Version 11.2.0
+<dl>
+
+<dt>Connection config "operation_timeout_ms"</dt>
```

Comment:
You use `config` here and `configuration` line 18, we should be consistent. I prefer the full word as it is a user-facing doc.

Reply:
Good catch. Fixed in [9b744b1](https://github.com/wiredtiger/wiredtiger/pull/9724/commits/9b744b1dd34097de332f27237d6ab56683a0af6b)